tests/tests/buy_and_sell_bitcoin_e2e_cnd_nectar.ts

```diff
@@ -28,8 +28,8 @@ test(
 
         await alice.assertBalancesAfterSwap();
         await bob.assertBalancesChangedBy({
-            bitcoin: -(9990725n + 3060n), // nectar pays order quantity + the funding fee
-            dai: 944_123_512_500_000_000_000n, // = 0.09990725 * 9450 * 10^18
+            bitcoin: -(10_000_000n + 3060n), // nectar pays order quantity + the funding fee
+            dai: 945_000_000_000_000_000_000n, // = 0.1 * 9450 * 10^18
         });
```

Comment:
Nice! :heart_eyes: 

Why doesn't the other test need changing?

Reply:
The other test did not need changes because the buy `quantity` is not having positions after the point because of fees - we don't deal with fees for buy - but because `max_sell` is in DAI. The `quantity` of the order is calculated out of the fixed DAI and the price (rate in Nectar), hence it fluctuates with the price. 
I thought about creating a follow-up PR where I change the `max` to being always in the base, i.e. `max_sell` and `max_buy` both being a Bitcoin quantity. Given the current model that makes more sense I feel.